RPGManager_development/src/rpg_database/character_sheet/Languages.java

```diff
@@ -0,0 +1,26 @@
+package rpg_database.character_sheet;
+
+import java.security.InvalidParameterException;
+import java.util.AbstractCollection;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import rpg_database.character_sheet.interfaces.CustomSetter;
+import rpg_database.character_sheet.interfaces.MultipleFieldsGetterSetter;
+
+public enum Languages {
+	ANDER("Ander"), ANTIVAN("Antivan"), CIRIANE("Ciriane"), TRADE_TONGUE("Trade tongue"), DWARVEN_LANGUAGES("Dwarven languages"),
+	ELVISH("Elvish"), QUNLAT("Qunlat"), ORLESIAN("Orlesian"), RIVAINI("Rivaini"), TEVENE("Tevene"), ANCIENT_TEVENE("Ancient Tevene");

```

Comment:
Érdekes, ezt nem is néztem... Majd jól jön RPGhez.
Minden esetre azokat használd légyszi amik ebben vannak: https://www.dropbox.com/s/97viilhz6ya9zmx/Dragonage%20Core%20Rulebook.pdf?dl=0